.semaphore/semaphore.yml

```diff
@@ -64,12 +64,12 @@ blocks:
           matrix:
             - env_var: CMD
               values:
-                - .ci/validation.sh no-error-orekit
```

Comment:
actually error is kind of the higher instance, therefore i would tend to rename all to error, if there is not real difference between them code wise.